modules/terminate-agent-hook/versions.tf

```
@@ -3,7 +3,7 @@ terraform {
 
   required_providers {
     aws = {
-      version = ">= 4.0.0"
+      version = ">= 5.78"
```

question: Why is this update necessary?

---

Just found the section in the docs of the provider. Seems to be version 5.76.0, right?

```suggestion
      version = ">= 5.76"
```